README.md

```diff
@@ -1,20 +1,236 @@
 
-## Note: This API is currently in private beta.
+FullStory Data Layer Observer (DLO) is a small but powerful JavaScript utility that makes integrating analytics data easier.  Through a flexible syntax and rules-driven approach, you can reference data from a data layer, perform intermediate changes, and send the result to FullStory - all without writing any custom JavaScript.
 
-An API for recording analytics data layer state into FullStory
+## Quick Start
 
-## Examples
+To experiment with DLO, begin by reviewing some examples.
+Something that describes getting started quickly and links to pre-built rule sets (e.g. CEDDL).
 
-The `fullstory-data-layer-observer/examples/` folder holds a variety of samples of the Data Layer Observer in action.
+## Viewing a Data Layer
 
-To run these examples:
+Before using DLO, it’s important to understand what’s in your data layer.  This can be as simple as visiting a web page in a browser like Chrome and opening the [JavaScript Console](https://developers.google.com/web/tools/chrome-devtools/console#javascript) (`option` + `cmd` + `j`).  Try typing `digitalData` or `dataLayer` or `s` in the text field next to the caret and hit `return`.  If something is printed that does not look like an error, that’s the data layer.  You can click the small triangle twistie to further explore what’s in the data layer.
 
-	cd fullstory-data-layer-observer/
-	npm install # you only need to do this once
-	npm run examples
+More sophisticated websites will often have a dictionary of what’s in a data layer and the meaning of the data.  Always consult your analytics team before using data found in a data layer.
 
-Now point your browser at http://127.0.0.1:8080/examples/
+## Data Privacy
 
+Sensitive, private, and confidential information should never be added to a data layer.  Various 3rd party libraries reside on most websites and have access to such information.  DLO uses the following approaches to additionally prevent unwanted data from being transmitted.
+
+- A rule-based design requires an admin to be intentional about data recording.  An admin must create and load a rule to collect any information from a data layer.  Recording is not automatic.
+- Despite being intentional, it’s possible that domain teams may later add unwanted properties to a previously approved object in the data layer.  DLO’s selector syntax allows an admin to specify which properties within an object are included or omitted as well as filtering techniques to further prevent recording unwanted data.
+
+## Deployment
+
+DLO is a JavaScript asset that is included on a web page.  FullStory hosts versions of DLO on our CDN.  Versioned releases have the naming convention `dlo-<version>.js`, and the most recent version is named `dlo-latest.js`.  If you would like the most update to date version of DLO on your site always, use `dlo-latest.js`.  If you'd rather stable releases and perform manual upgrades, use `dlo-<version>.js`.
```

Comment:
You could also mention that we're using SemVer and link to our Changelog.md for folks who want more details about version changes